### PR TITLE
Add 5.17.2 release notes and add 5.17.2 to supported versions page

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Adds release notes for 5.17.2
- Adds 5.17.2 to supported versions page

## Motivation and Context
Support 5.17.2 release

### CODE REVIEW

File: content/sensu-go/5.17/release-notes.md [hunk 1]
```diff
@@ -52,6 +53,17 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.17.2 release notes
+
+**February 19, 2020** &mdash; The latest release of Sensu Go, version 5.17.2, is now available for download.
+This release fixes a bug that could cause enterprise HTTP routes to fail to initialize upon internal restart.
```
Comment (palourde) [inline content/sensu-go/5.17/release-notes.md]: I actually have no clue but do we prefer to call them _enterprise_ or _commercial_ routes now?

Comment (hillaryfraley) [reply content/sensu-go/5.17/release-notes.md]: You are correct -- it should be "commercial" and thank you for catching this!

Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
This release fixes a bug that could prevent commercial features from working after internal restart.
```

File: content/sensu-go/5.17/release-notes.md [hunk 2]
```diff
@@ -52,6 +53,17 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.17.2.
+
+**FIXES:**
+
+- Fixed a bug that could cause enterprise HTTP routes to fail to initialize on an internal restart.
```
Comment (hillaryfraley) [inline content/sensu-go/5.17/release-notes.md]:
```suggestion
- Fixed a bug that could cause commercial HTTP routes to fail to initialize after an internal restart, preventing commercial features from working.
```

**Pull request merged**